docs/How-to-release.md

```diff
@@ -1,34 +1,45 @@
-# Apache SkyWalking release guide
+# Apache SkyWalking CLI release guide
 
-This documentation guides the release manager to release the SkyWalking CLI in the Apache Way, and also helps people to
-check the release for vote.
+This documentation guides the release manager to release the SkyWalking CLI in the Apache Way, and also helps people to check the release for vote.
 
-## Add your GPG public key
+## Prerequisites
 
-1. Add your GPG public key into [SkyWalking GPG KEYS](https://dist.apache.org/repos/dist/release/skywalking/KEYS) file,
-you can do this only if you are a committer, use your Apache id and password to log into the svn, and update the file.
-**DO NOT override the existed `KEYS` file.**
+1. Clear all milestone issues from [skywalking-cli](https://github.com/apache/skywalking-cli/milestones) and [skywalking](https://github.com/apache/skywalking/milestones), create a new one if needed.
```

Comment:
```suggestion
1. Close(if finished, or move to next milestone otherwise) all issues in the current milestone from [skywalking-cli](https://github.com/apache/skywalking-cli/milestones) and [skywalking](https://github.com/apache/skywalking/milestones), create a new milestone if needed.
```